tox.ini

```diff
@@ -10,11 +10,14 @@ envdir =
     py37: {toxworkdir}/3.7
     py38: {toxworkdir}/3.8
     py39: {toxworkdir}/3.9
+    py310: {toxworkdir}/3.10
     pypy3: {toxworkdir}/pypy3
     py: {toxworkdir}/py
 commands =
     python housekeep.py prep
-    !diffcov: bandit -c bandit.yml -r aiosmtpd
+    # Bandit is not needed on diffcov, and seems to be incompatible with 310
+    # So, run only if "not (310 or diffcov)" ==> "(not 310) and (not diffcov)"
+    !py310-!diffcov: bandit -c bandit.yml -r aiosmtpd
```

Comment:
In `tox` logic, the 'and' operator <kbd>-</kbd> bounds less tightly than the 'no' operator <kbd>!</kbd>